wfapi/wfapi.ipldsch

```diff
@@ -430,4 +430,27 @@ type CatalogMirrorsByPacktype {Packtype:WarehouseAddrList}
 
 type ReplayCapsule union {
 	| Plot "plot.v1"
-} representation keyed
\ No newline at end of file
+} representation keyed
+
+# MirroringConfig defines the mirroring configuration for an entire workspace.
+# This maps the addresses which we want to publish to (of type WarehouseAddr) 
+# to specific configs, which may be backed by various types of storage.
```

Comment:
Could use some discussion about the prefixing logic, and some examples.

(There aren't really well established formatting rules for these comments, but I think using indentation for example content, roughly like godoc does, is probably fine and well.)